Engine/Setup/BaseSetupHandler.cs

```diff
@@ -35,6 +36,12 @@ namespace QuantConnect.Lean.Engine.Setup
     /// </summary>
     public static class BaseSetupHandler
     {
+        /// <summary>
+        /// Get the maximum time that the creation of an algorithm can take
+        /// </summary>
+        public static TimeSpan AlgorithmCreationTimeout =>
+            TimeSpan.FromSeconds(Config.GetDouble("algorithm-creation-timeout", 90));
```

Comment:
Could directly be `GetInt`?

Reply:
I used `GetDouble` because `[TimeSpan.FromSeconds](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.fromseconds?view=net-5.0)` takes a double.